### ISSUE TITLE
Doesn't work with cordova-plugin-crosswalk-webview

### ISSUE THREAD

**Issue by jfrumar** [phonegap/phonegap-mobile-accessibility#32]
This plugin appears to be working on Android with the system WebView, however fails when using a [Crosswalk](https://github.com/crosswalk-project/cordova-plugin-crosswalk-webview) webview.

It looks like [casting a CordovaWebViewEngine.EngineView to a WebView](https://github.com/phonegap/phonegap-mobile-accessibility/blob/master/src%2Fandroid%2Fcom%2Fphonegap%2Fplugin%2Fmobileaccessibility%2FDonutMobileAccessibilityHelper.java#L51) is not possible and results in an exception.

**Comment by majornista** [comment]
@jfrumar Give it a try now to see if using reflection for `getView`, `getSettings`, `getTextSize`/`getTextZoom`, and `setTextSize`/`setTextZoom` so that the View does not need to be cast as an `android.webkit.WebView` allows plugin to be compatible with `XWalkView` and `XWalkSettingsInternal`.

**Comment by hkvstore** [comment]
Have the same problem with CrossWalk, It seems that the changes still do not solve the problem.

**Comment by majornista** [comment]
@hkvstore and @jfrumar Is my change still causing an uncaught exception or is it a problem of Crosswalk not supporting `getTextZoom` and `setTextZoom` methods. If it's the later, I may need to refactor the `getTextZoom` and `setTextZoom` methods to use `getResources().getConfiguration().fontScale`.

**Comment by hkvstore** [comment]
>is it a problem of CrossWalk not supporting getTextZoom and setTextZoom methods.

I cannot say for sure, I tested on a real Android phone only, there is no error when calling setTextZoom(), but I suspect the CrossWalk plugin does not support getSettings().

**Comment by majornista** [comment]
Looks like getSettings() has been added in [Version 6 - Crosswalk 17](https://crosswalk-project.org/apis/embeddingapidocs_v6/index.html), however it doesn't seem to do anything to the text zoom setting within Crosswalk's Chromium instance.

**Comment by JeremyColton** [comment]
Hi,
I'm using Cordova 5.0,0, Android 6, CrossWalk 2.0.0, running on Samsung S4 and I get this error:

09-27 23:40:10.951: E/AndroidRuntime(13802): java.lang.RuntimeException: Unable to start activity ComponentInfo{com.scoochup/com.scoochup.ScoochUpMobileApp500_Android}: java.lang.ClassCastException: org.crosswalk.engine.XWalkCordovaView cannot be cast to android.webkit.WebView

I don't see errors in my project in Eclipse since I use the command line to build the project via: 
>cordova android run

So how can I fix this?

**Comment by majornista** [comment]
@JeremyColton I don't think that you can cast the org.crosswalk.engine.XWalkCordovaView as a android.webkit.WebView. You can see how I work around this here: https://github.com/phonegap/phonegap-mobile-accessibility/blob/master/src/android/com/phonegap/plugin/mobileaccessibility/MobileAccessibility.java#L139
